gluetool/glue.py

```diff
@@ -1905,7 +1905,14 @@ def module_config_paths(self):
         """
 
         from .utils import normalize_path_option
-        return normalize_path_option(self.option('module-config-path')) or DEFAULT_MODULE_CONFIG_PATHS
+
+        config_paths = DEFAULT_MODULE_CONFIG_PATHS
```

Comment:
`config_paths` is set just to be thrown away if the option is set. How about this?

```python
if self.option('module-config-path'):
    return normalize_path_option(self.option('module-config-path'))

if 'GLUETOOL_MODULE_CONFIG_PATHS' in os.environ:
  return normalize_path_option(os.environ['GLUETOOL_MODULE_CONFIG_PATHS'])

return DEFAULT_MODULE_CONFIG_PATHS
```

Reply:
looks better :) thanks, done